lib/new_relic/error/metadata_reporter.ex

```diff
@@ -81,6 +81,7 @@ defmodule NewRelic.Error.MetadataReporter do
 
   defp parse_process_name([], [{module, _f, _a, _} | _]), do: inspect(module)
   defp parse_process_name([], _stacktrace), do: "UnknownProcess"
```

Comment:
just noting out loud i'm curious about using "UnknownProcess" here -- seems bubbling up a nil would be better for later querying